python/metatomic_torch/tests/ase_calculator.py

```diff
@@ -549,3 +551,73 @@ def test_additional_outputs(atoms):
 
     another = atoms.calc.additional_outputs["another::one"]
     assert another.block().properties.names == ["another"]
+
+
+def with_isolated_imports(test_func):
```

Comment:
IMO not great to have a decorator here instead of in `configtest` but it also has zero use anywhere else.